docs/src/rules/generator-star.md

```diff
@@ -6,7 +6,6 @@ further_reading:
 - https://leanpub.com/understandinges6/read/#leanpub-auto-generators
 ---
 
-Enforces consistent spacing around the asterisk in generator functions.
```

Comment:
I think we should keep descriptions in the docs for removed rules because they don't have descriptions from rules meta.

Reply:
Can you also restore descriptions in other removed rules? You can find the list of removed rules here:

https://github.com/eslint/eslint/blob/6840940a766d671831d5cee0e3c0e2f4e642632a/conf/rule-type-list.json#L12-L35

Reply:
Restored descriptions for removed rules.